src/arch/arm/config.cmake

```diff
@@ -32,6 +32,11 @@ elseif(KernelArmCortexA72)
     # (https://developer.arm.com/documentation/100095/0001/memory-management-unit/about-the-mmu)
     set(KernelArmPASizeBits44 ON)
     math(EXPR KernelPaddrUserTop "(1 << 44)")
+elseif(KernelArmCortexA78AE)
+    # Even though CortexA78AE supports 48-bits
+    # 44 bits PA is used
```

Comment:
This seems Oring specific then? So the comment should clearly say this, or we could even have a platform-specific setting then for Orin besides the A78AE's default

Reply:
Ah, that should be more clear, while the A78AE supports 48 bits, I haven't figured out a way to enabke this without causing a chain reaction of assertion failures within the kernel.

My understanding is that the ARM64 kernel only has _47_ useable bits of physical memory unless hypervisor mode is enabled, if anyone knows the context to this please share :).

Reply:
Considering the Jetson Orinmodules don't support more than 64GB, let alone 1TB, why not treat is as a Cortex-A78?

Or are you also adding lockstep support to seL4 later on?

Reply:
There is some discussion on 48 bit physical address space in https://github.com/seL4/seL4/issues/1175 and https://github.com/seL4/seL4/pull/1157, but the result is basically that the full 48 bit are currently not supported and would require further changes.